### PR TITLE
[GK 90, 122, 101, 113, 114, 115] status websocket, kontrola meczu

### PR DESCRIPTION
**Pull request by RysiekPrecikowski**

### CODE REVIEW

**Review by peep-hole — APPROVED** [review]

File: backend/src/main/java/pl/edu/agh/MatchStatusManager.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+import java.util.UUID;
+
+@Component
+@Getter
+@Setter
+@ToString
+@RequiredArgsConstructor
+public class MatchStatusManager {
+    private final MatchManagementService matchManagementService;
+    Map<UUID, Match> matchMap = new HashMap<>();
```
Comment (ajablonsk1) [inline backend/src/main/java/pl/edu/agh/MatchStatusManager.java]: private final

File: backend/src/main/java/pl/edu/agh/MatchStatusManager.java [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+            if (message.getRefereeToken().equals(match.getSideRefereeToken1())) {
+                decision = match.getReferee1Decision();
+            } else if (message.getRefereeToken().equals(match.getSideRefereeToken2())) {
+                decision = match.getReferee2Decision();
+            }
+            else {
+                throw new RuntimeException("WRONG REFEREE ID");
+            }
+
+            decision.points1 = message.getPoints1();
```
Comment (ajablonsk1) [inline backend/src/main/java/pl/edu/agh/MatchStatusManager.java]: You got setters and getters, use it. For example decision.setPoints1(message.getPoints1()). It is unlikely to use this: decision.points1 (becouse of encapsulation)

File: backend/src/main/java/pl/edu/agh/SideRefereeDecision.java [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+import lombok.Getter;
+import lombok.Setter;
+import pl.edu.agh.constants.Event;
+
+import java.util.List;
+
+@Getter
+@Setter
+public class SideRefereeDecision {
+    int points1;
```
Comment (ajablonsk1) [inline backend/src/main/java/pl/edu/agh/SideRefereeDecision.java]: private

File: backend/src/main/java/pl/edu/agh/SideRefereeDecision.java [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+    int points2;
+
+    List<Event> decision;
```
Comment (ajablonsk1) [inline backend/src/main/java/pl/edu/agh/SideRefereeDecision.java]: private final

File: backend/src/main/java/pl/edu/agh/websocket/DecisionMessage.java [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+import java.util.List;
+import java.util.UUID;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@ToString
+public class DecisionMessage {
+    int points1;
```
Comment (ajablonsk1) [inline backend/src/main/java/pl/edu/agh/websocket/DecisionMessage.java]: private

File: backend/src/main/java/pl/edu/agh/websocket/DecisionMessage.java [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+    int points2;
+
+    List<Event> decision;
```
Comment (ajablonsk1) [inline backend/src/main/java/pl/edu/agh/websocket/DecisionMessage.java]: private final

File: backend/src/main/java/pl/edu/agh/websocket/DecisionMessage.java [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+    UUID refereeToken;
```
Comment (ajablonsk1) [inline backend/src/main/java/pl/edu/agh/websocket/DecisionMessage.java]: private final (I guess it is final)

**Review by ajablonsk1 — COMMENTED** [review]
Looks good, just remember about data encapsulation (private modifier).

**Pull request merged**